ghwatch.py

```diff
@@ -195,7 +195,17 @@ def get_html_url(ghbase, rec):
 def priority_sort_key(item):
     return (-REASON_PRIO.index(item.reason), item.updated_at)
 
-def draw():
+def github_load(user):
+    since = datetime.datetime.utcnow() - datetime.timedelta(days=args.days)
+    get_all = True if args.all else GithubObject.NotSet
+    issues = list(user.get_notifications(all=get_all, since=since))
```

Comment:
btw i only just noticed (this was already the case) but it would be better to call this `notifications` than `issues`

Reply:
Oops.. yeah, you're right :D